docs/resources/budget.md

```diff
@@ -0,0 +1,106 @@
+---
+subcategory: "Deployment"
+---
+# databricks_mws_budget Resource
+
+-> **Note** Initialize provider with `alias = "account"`, `host = "https://accounts.azuredatabricks.net"` and use `provider = databricks.account` for all `databricks_mws_*` resources.
```

Comment:
We don't have budgets on Azure yet.

Reply:
Changed to the AWS equivalent `https://accounts.cloud.databricks.com`